docs/content/usage/configuration/basic-network.md

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
I don't know if we should have this page in prod. This adds nothing that config page doesn't explain

Reply:
I'd rather leave this section online and wait for your config PR - then we could improve this page (with example values and such)

Reply:
why not push it as draft?